.github/workflows/pr-label.yml

```diff
@@ -34,7 +36,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: codelytv/pr-size-labeler@v1.7.0
+      - uses: codelytv/pr-size-labeler@v1
```

Comment:
Undo this.  v1 version has a bug. add a note to keep specifically v1.7.0

Reply:
According to these comments, it's a matter of setting the right permissions and not having the wrong API URL in the config:
https://github.com/CodelyTV/pr-size-labeler/issues/27#issuecomment-1102806189
https://github.com/CodelyTV/pr-size-labeler/issues/40#issuecomment-1142130933
https://github.com/CodelyTV/pr-size-labeler/issues/40#issuecomment-1342583471

The workflow as it is in this PR should work.